README.md

```diff
@@ -3,8 +3,10 @@
 
 Avatar Animator is a fork of [Pose Animator](https://github.com/yemount/pose-animator), a fast and real-time 2D vector illustrations based on Google's [PoseNet](https://github.com/tensorflow/tfjs-models/tree/master/posenet) and [FaceMesh](https://github.com/tensorflow/tfjs-models/tree/master/facemesh) model. It wraps Pose Animator in an Electron app and sends it to a virtual camera for use in video-conferencing apps like Zoom, Microsoft Teams, and Google Meet.
```

Comment:
```suggestion
Avatar Animator is a lightweight tool for generating real-time 2D vector-based avatar animations using webcams or other video devices. It can be used with Zoom, Skype, Microsoft Team, Google Meet, and many more video-conferencing apps. The project is a fork of [Pose Animator](https://github.com/yemount/pose-animator), a fast and real-time 2D vector illustrations animator based on Google's [PoseNet](https://github.com/tensorflow/tfjs-models/tree/master/posenet) and [FaceMesh](https://github.com/tensorflow/tfjs-models/tree/master/facemesh) model. It wraps Pose Animator in an Electron app and sends it to a virtual camera.
```